src/Serialise/Cborg/FlatTerm.hs-boot

```diff
@@ -0,0 +1,10 @@
+module Serialise.Cborg.FlatTerm where
+
+import {-# SOURCE #-} Serialise.Cborg.Encoding
+
+type FlatTerm = [TermToken]
+
+data TermToken
+instance Show TermToken
+
+toFlatTerm :: Encoding -> FlatTerm
```

Comment:
Not sure if anyone cares, but this (and a few other files) don't end with newlines